tests/middleware/test_base.py

```diff
@@ -468,7 +468,7 @@ async def cancel_on_disconnect(
 
             # A timeout is set for 0.1 second in order to ensure that
             # we never deadlock the test run in an infinite loop
-            with anyio.move_on_after(0.1):
+            with anyio.move_on_after(0.2):
```

Comment:
Why?

Reply:
BlockBuster has a small toll on performance.
In CI, the test machines are already slow, the cancellation sometimes arrives too late and the test is flaky.